### PR TITLE
Fix multiple filtering bug in https://github.com/participedia/usersnaps/issues/960

### PR DESCRIPTION
**Pull request by zphingphong**

### CODE REVIEW

File: api/helpers/shared-field-options.js [hunk 1]
```diff
@@ -470,4 +470,5 @@ module.exports = {
     "partial_editing",
     "complete",
   ],
+  query: []
```
Comment (ascott) [inline api/helpers/shared-field-options.js]: i'm not sure about this solution. this file is for storing the option keys for fields, and `query` isn't a field. i'm wondering if there is another solution here?

**Comment by ascott** [comment]
moving your comment from the issue here for discussion: 
> The bug is

> All query parameter filter are in shared-field-option.js.
> Now, if there are multiple query. The filtering logic happened in `handlebars-helpers.js@filterCollections()`.

> And the error shows up because the query parameter query doesn't exist in shared-field-option.js.

> To fix this I added a new item in the array for query to handle this line in shared-field-option.js
> `if ( sharedFieldOptions[keys[i]].includes(arr[x]) ) {`

can you edit `handlebars-helpers.js@filterCollections()` to handle the case when the `query` param is present rather than adding it as a field in `shared-field-option.js`?

File: api/helpers/shared-field-options.js (hunk 1 above)
Comment (zphingphong) [reply api/helpers/shared-field-options.js]: @ascott thanks! how about this solution? https://github.com/participedia/api/pull/824/commits/43d3b5f4d5ff5c4d66eab032cc9dd4f601059aaf

**Comment by ascott** [comment]
thanks for the change @zphingphong, i think this makes more sense! :+1:

**Pull request merged**